index.html

```diff
@@ -4,9 +4,9 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <title>Notes app</title>
   </head>
-  <body>
+  <body class="bg-primary-light">
     <div id="root"></div>
     <script type="module" src="/src/main.jsx"></script>
```

Comment:
I am using it to set background-color of the body ( the whole page). To avoid confusion I am going to move it into `index.css`  to have all styles in one place